Menu documentation is out of date

The [menu documentation](https://github.com/atom/electron/blob/master/docs/api/menu.md) gives an example of creating the application menu with the simple template API, with the comment that the code belongs in `main.js`.

However, the code doesn't work there - it gives this error:

```
Uncaught Exception:
ReferenceError: Menu is not defined
    at EventEmitter.<anonymous> (/Users/alexwarren/Documents/Projects/squiffy-editor/main.js:171:14)
    at emitOne (events.js:82:20)
    at EventEmitter.emit (events.js:166:7)
```

Adding these lines from the `index.html` example doesn't help either:

    var remote = require('remote');
    var Menu = remote.require('menu');

... because then you get:

```
Error: Cannot find module 'remote'
```

It *does* work if you place it in `index.html`... in as much as the menus are created. **But** then the "Reload" and "Toggle DevTools" menus don't work, because they require a reference to `BrowserWindow`.

--- a/docs/api/menu.md
+++ b/docs/api/menu.md
@@ -30,7 +30,9 @@ window.addEventListener('contextmenu', function (e) {
 Another example of creating the application menu with the simple template API:
 
 ```javascript
-// main.js
+<!-- index.html -->
+var remote = require('remote');
+var Menu = remote.require('menu');
 var template = [
   {
     label: 'Electron',
@@ -69,7 +71,7 @@ var template = [
       {
         label: 'Quit',
         accelerator: 'Command+Q',
-        click: function() { app.quit(); }
+        selector: 'terminate:'
       },
     ]
   },
@@ -117,12 +119,12 @@ var template = [
       {
         label: 'Reload',
         accelerator: 'Command+R',
-        click: function() { BrowserWindow.getFocusedWindow().reloadIgnoringCache(); }
+        click: function() { remote.getCurrentWindow().reloadIgnoringCache(); }
       },
       {
         label: 'Toggle DevTools',
         accelerator: 'Alt+Command+I',
-        click: function() { BrowserWindow.getFocusedWindow().toggleDevTools(); }
+        click: function() { remote.getCurrentWindow().toggleDevTools(); }
       },
     ]
   },
@@ -156,7 +158,7 @@ var template = [
 
 menu = Menu.buildFromTemplate(template);
 
-Menu.setApplicationMenu(menu); // Must be called within app.on('ready', function(){ ... });
+Menu.setApplicationMenu(menu);
 ```
 
 ## Class: Menu

--- a/docs/api/menu.md
+++ b/docs/api/menu.md
@@ -29,8 +29,9 @@ window.addEventListener('contextmenu', function (e) {
 
 Another example of creating the application menu with the simple template API:
 
-```javascript
+```html
 <!-- index.html -->
+<script>
 var remote = require('remote');
 var Menu = remote.require('menu');
 var template = [
@@ -159,6 +160,7 @@ var template = [
 menu = Menu.buildFromTemplate(template);
 
 Menu.setApplicationMenu(menu);
+</script>
 ```
 
 ## Class: Menu

